.github/workflows/test.yml

```
@@ -0,0 +1,24 @@
+name: Run Vitest on PR and Main Push
+
+on:
+  pull_request:
+  push:
```

지훈님 브랜치에 대해 동작이 일어날 때 워크플로우가 돌아야하니, 아래처럼 작성해줄 수 있겠네요 😊

```suggestion
  pull_request:
    branches: [hoon-ground]
  push:
    branches: [hoon-ground]
```